app/services/customers/manage_invoice_custom_sections_service.rb

```diff
@@ -2,27 +2,30 @@
 
 module Customers
   class ManageInvoiceCustomSectionsService < BaseService
-    def initialize(customer:, skip_invoice_custom_sections:, section_ids:)
+    def initialize(customer:, skip_invoice_custom_sections:, section_ids: nil, section_codes: nil)
       @customer = customer
       @section_ids = section_ids
+      @section_codes = section_codes
       @skip_invoice_custom_sections = skip_invoice_custom_sections
 
       super
     end
 
     def call
       return result.not_found_failure!(resource: "customer") unless customer
-      raise_invalid_params if skip_invoice_custom_sections && section_ids.present?
+      raise_double_selection if !section_ids.nil? && !section_codes.nil?
```

Comment:
You might need to return the result here (or am I missing something?)

Reply:
oops, yes, thanks